src/main/java/com/wiley/autotest/actions/RepeatableAction.java

```diff
@@ -79,6 +93,22 @@ public void perform() {
         }
     }
 
+    public T performAndGet() {
```

Comment:
whenever we need to add "and" in a name - it might be a sign that something might be wrong with our design